src/server/ua_subscription.c

```diff
@@ -19,6 +19,106 @@
 
 #ifdef UA_ENABLE_SUBSCRIPTIONS /* conditional compilation */
 
+UA_MonitoredItem *
```

Comment:
Can we keep these functions in `ua_subscription_datachange.c`? They nearly did not change.

We should keep subscriptions separate from monitoreditems where possible.
The file can be renamed to `ua_subscription_monitoreditems.c` when events drop.

Reply:
The idea was to have an additional file `ua_subscription_events.c` in which the functions for the events would be. The functions were moved to `ua_subscription.c` since monitoredItems are also defined in `ua_subscription.h`. Can of course still put them back into `ua_subscription_datachange.c`.